charts/incubator/piwigo/values.yaml

```
@@ -35,10 +35,14 @@ workload:
                 - /bin/sh
                 - -c
                 - |
-                  echo "sending api request for initial piwigo setup"
-                  echo "if set previously, piwigo will disregard request"
-                  curl -X POST -d "language={{ .Values.piwigo.language }}&dbhost={{ .Values.mariadb.creds.plainhost | trimAll "\"" }}:3306&dbuser={{ .Values.mariadb.mariadbUsername }}&dbpasswd={{ .Values.mariadb.creds.mariadbPassword | trimAll "\"" }}&dbname={{ .Values.mariadb.mariadbDatabase }}&prefix=piwigo_&admin_name={{ .Values.piwigo.admin_user }}&admin_pass1={{ .Values.piwigo.admin_pass }}&admin_pass2={{ .Values.piwigo.admin_pass }}&admin_mail={{ .Values.piwigo.admin_email }}&install=Start+installation" http://0.0.0.0/install.php
-                  echo "done"
+                  post_setup="/config/POSTSETUP"
+                  if [ -f "$post_setup" ]; then
+                    echo "POSTSETUP exists, no further action is needed."
+                  else
+                    echo "sending api request for initial piwigo setup"
+                    curl -X POST -d "language={{ .Values.piwigo.language }}&dbhost={{ .Values.mariadb.creds.plainhost | trimAll "\"" }}:3306&dbuser={{ .Values.mariadb.mariadbUsername }}&dbpasswd={{ .Values.mariadb.creds.mariadbPassword | trimAll "\"" }}&dbname={{ .Values.mariadb.mariadbDatabase }}&prefix=piwigo_&admin_name={{ .Values.piwigo.admin_user }}&admin_pass1={{ .Values.piwigo.admin_pass }}&admin_pass2={{ .Values.piwigo.admin_pass }}&admin_mail={{ .Values.piwigo.admin_email }}&install=Start+installation" http://0.0.0.0/install.php
+                    touch "$post_setup" && echo "Created POSTSETUP!" || echo "Failed to create POSTSETUP file..."
```

```suggestion
                    curl -X POST -d "language={{ .Values.piwigo.language }}&dbhost={{ .Values.mariadb.creds.plainhost | trimAll "\"" }}:3306&dbuser={{ .Values.mariadb.mariadbUsername }}&dbpasswd={{ .Values.mariadb.creds.mariadbPassword | trimAll "\"" }}&dbname={{ .Values.mariadb.mariadbDatabase }}&prefix=piwigo_&admin_name={{ .Values.piwigo.admin_user }}&admin_pass1={{ .Values.piwigo.admin_pass }}&admin_pass2={{ .Values.piwigo.admin_pass }}&admin_mail={{ .Values.piwigo.admin_email }}&install=Start+installation" http://0.0.0.0/install.php && ( touch "$post_setup" && echo "Created POSTSETUP!" || echo "Failed to create POSTSETUP file..." )
```
Something like this, to ensure the file isn't made unless the command is executed successfully.
Also check the existcode from curl, it needs to exist non-0 when it fails to execute the command